src/containerapp/azext_containerapp/_up_utils.py

```diff
@@ -409,7 +409,10 @@ def __init__(
         env_vars=None,
         workload_profile_name=None,
         ingress=None,
-        force_single_container_updates=None
+        force_single_container_updates=None,
+        registry_identity=None,
```

Comment:
should this parameter be exposed?

Reply:
For az containerapp up, we expose new arguments `--registry-identity`, `--system-assigned`, `--user-assigned`

Reply:
I do not see --registry-identity added in params.py

Reply:
For --registry-identity, it is Inherited from azure-cli `az containerapp`:
![image](https://github.com/user-attachments/assets/b96cc598-f0fb-4fbb-9800-439152e58af1)

But for --system-assigned, --user-assigned I cannot find where it Inherited from, but with test we can see the decription in --help.
![image](https://github.com/user-attachments/assets/680fcfd3-434f-497a-853f-4ab377785862)